registry/registry.go

```diff
@@ -345,7 +345,7 @@ func configureLogging(ctx context.Context, config *configuration.Configuration)
 	logrus.SetLevel(logLevel(config.Log.Level))
 
 	formatter := config.Log.Formatter
-	if formatter == "" {
+	if formatter == "" { //just let the library use default on empty string.
 		formatter = "text" // default formatter
```

Comment:
Not sure if the extra comment adds much, as it's basically repeating the comment immediately underneath it.

Perhaps we should combine this condition in the switch though.